spec/models/social_networking/profile_answer_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "spec_helper"
+
+module SocialNetworking
+  describe ProfileAnswer, type: :model do
```

Comment:
I think if you want to test that the validation exists, it should be a pretty minimal test. Rather than these 15 lines, what about reusing the fixtures with something like:

```ruby
existing_answer = ProfileAnswer.first
new_answer = ProfileAnswer.new(profile_id: existing_answer.profile_id,
                               profile_question_id: existing_answer.profile_question_id).tap(&:valid?)

expect(new_answer.errors[:profile_question_id].size).to eq 1
expect(new_answer.errors[:profile_question_id][0]).to eq "has already been answered"
```